src/System.Runtime/tests/Performance/Perf.Type.cs

```diff
@@ -5,36 +5,44 @@
 
 namespace System.Runtime.Tests
 {
-    public class Perf_Type
+    public static class Perf_Type
     {
         [Benchmark]
-        public void GetTypeFromHandle()
+        public static void GetTypeFromHandle()
         {
-            RuntimeTypeHandle type1 = typeof(int).TypeHandle;
+            RuntimeTypeHandle th = typeof(int).TypeHandle;
```

Comment:
Same question here on the naming change - 'type' or 'typeHandle' are a lot better names than 'th' IMO.